.github/workflows/udpdate.yml

```diff
@@ -17,16 +17,34 @@ jobs:
     - name: "Skip if no new version"
       uses: Geode-solutions/actions/need-update@master
       id: update
-    - name: Stop
-      if: !steps.update.outputs.update
-      run: exit 0
+      with:
```

Comment:
You may a merge. I hope you did not change this file

Reply:
I think i created this branch from the develop branch and not the master, that's why

Reply:
So update from master before to avoid strange diff to review

Reply:
That's what I did